src/argilla/client/feedback/dataset/local/dataset.py

```diff
@@ -162,13 +163,21 @@ def records(self) -> List["FeedbackRecord"]:
 
     def __repr__(self) -> str:
         """Returns a string representation of the dataset."""
-        return (
```

Comment:
@kursathalat we should just use this old default formatting but add `metadata_properties` to it.